components/util/useCopy.tsx

```diff
@@ -0,0 +1,26 @@
+import { useEffect, useState } from "react";
```

Comment:
Where do you guys think we should put custom hooks? Not sure this is the best/most obvious place.

Reply:
I agree that the util folder probably isn't quite the right place. In the demo app, we had them on them in a hooks folder on the same level as the components folder, but I could also see putting them in a hooks folder inside the components folder

Reply:
Ended up moving to `/hooks`